packages/lesswrong/server/analytics/postViewTimesHybridView.ts

```diff
@@ -0,0 +1,40 @@
+import { registerHybridAnalyticsView } from "./hybridViews";
+
+export const POST_VIEW_TIMES_IDENTIFIER = "post_view_times";
+
+const viewQuery = (crossoverTime: Date, materialized = false) => `
+  SELECT
+    client_id,
+    post_id,
+    (date_trunc('day', timestamp) + interval '1 second') AS window_start,
+    (date_trunc('day', timestamp) + interval '1 day') AS window_end,
+    sum(INCREMENT) AS reading_time
```

Comment:
What does this do? I'm not sure if this is related, but I tried to increase the # of reads for a post by staying on it for 30 seconds (I saw the timer event get logged), but the # of reads in the table didn't increase.

Reply:
This is query behind the view we use for reads, it gets the total reading time per [clientId, postId, day]. Then in `analyticsResolvers` we query for this number > 30s to get the number of reads (in `runReadsQuery`). It sounds like this is what you expected so it should have worked for you.

I just checked this myself and I was able to get the reads number to go up. The check is for `> 30s` rather than `>= 30s`, I wonder if that's why it didn't work for you?

Reply:
Yup I waited longer and it worked, thanks!